scripts/Makefile

```diff
@@ -22,8 +19,6 @@ install:
 	mkdir -p $(DESTDIR)/usr/lib/systemd/system
 	mkdir -p $(DESTDIR)/usr/lib/yum-plugins
 	mkdir -p $(DESTDIR)$(OPTDIR)/packages/post-install-scripts
-	mkdir -p $(DESTDIR)$(SITE_DIR)
-	mkdir -p $(DESTDIR)$(SITE3_DIR)
```

Comment:
Are we sure this is not needed anymore for py3 either?

Reply:
yes, it now goes to python3/Makefile.

Reply:
Correct, these directories are now created (and used for installs) only in `python3/Makefile`.

Not removing these `mkdirs` in `scripts/Makefile` much earlier was an oversight.